ssg/build_cpe.py

```diff
@@ -308,11 +327,10 @@ def extract_referred_nodes(tree_with_refs, tree_with_ids, attrname):
     reflist = []
     elementlist = []
 
-
     # decide on usage of .iter or .getiterator method of elementtree class.
     # getiterator is deprecated in Python 3.9, but iter is not available in
     # older versions
-    if getattr(tree_with_refs, "iter", None) == None:
+    if getattr(tree_with_refs, "iter", None) is None:
         tree_with_refs_iterator = tree_with_refs.getiterator()
     else:
         tree_with_refs_iterator = tree_with_refs.iter()
```

Comment:
This is used in multiple places, please extract, name accordingly, and then remove comments.

Reply:
Fixed: https://github.com/ComplianceAsCode/content/pull/8923/commits/be411916c581a7e0e17960487c0d10ea026b8a2a